op-test-sequencer/sequencer/backend/backend.go

```diff
@@ -54,20 +55,39 @@ func (ba *Backend) setupSequencerFrontend(id seqtypes.SequencerID) error {
 	if err := ba.router.AddRPC(route); err != nil {
 		return fmt.Errorf("invalid sequencer RPC route: %w", err)
 	}
-	f := &frontend.SequencerFrontend{Sequencer: ba.ensemble.Sequencer(id)}
+
+	tx_route := "/tx/" + id.String()
+	isAuthenticated := false
+	if err := ba.router.AddRPCWithAuthentication(tx_route, &isAuthenticated); err != nil {
+		return fmt.Errorf("invalid sequencer RPC route: %w", err)
+	}
+
+	f := &frontend.SequencerFrontend{Sequencer: ba.ensemble.Sequencer(id), Logger: ba.logger}
 	if err := ba.router.AddAPIToRPC(route, rpc.API{
 		Namespace: "sequencer",
 		Service:   f,
 	}); err != nil {
 		return fmt.Errorf("invalid sequencer RPC frontend: %w", err)
 	}
 	ba.logger.Info("Added sequencer RPC route", "route", route)
+
+	ethTx := &frontend.EthTxFrontend{Sequencer: ba.ensemble.Sequencer(id), Logger: ba.logger}
+	if err := ba.router.AddAPIToRPC(tx_route, rpc.API{
+		Namespace: "eth",
+		Service:   ethTx,
+	}); err != nil {
+		return fmt.Errorf("invalid ethTx RPC frontend: %w", err)
+	}
+	ba.logger.Info("Added ethTx RPC route", "route", tx_route)
```

Comment:
Technically the `eth_sendRawTransaction` within the `op-test-sequencer` is not used in this particular PR, but will be used in a follow-up test.